test/gui/blueprint.lua

```diff
@@ -58,7 +49,7 @@ function test.minimal_happy_path()
             expect.table_eq({'2', '3', '-4', 'blueprint', '--cursor=10,20,33'},
                             mock_run.call_args[1])
             send_keys('SELECT') -- dismiss the success messagebox
-            delay() -- allow the messagebox and view to disappear
+            while view:isActive() do delay() end -- wait for view to disappear
```

Comment:
This could potentially cause tests to hang indefinitely (disregarding CI timeouts, of course) if the screen being tested doesn't dismiss itself properly. Should we have a `delay_until()` helper that times out after some number of ticks?

Reply:
good idea. I'll spin a PR.

Reply:
done